src/NLog/Targets/ColoredConsoleTarget.cs

```diff
@@ -177,14 +185,31 @@ protected override void CloseTarget()
             base.CloseTarget();
         }
 
-            /// <summary>
+        /// <summary>
         /// Writes the specified log event to the console highlighting entries
         /// and words based on a set of defined rules.
         /// </summary>
         /// <param name="logEvent">Log event.</param>
         protected override void Write(LogEventInfo logEvent)
         {
-            this.Output(logEvent, this.Layout.Render(logEvent));
+            if (!FastModus)
+            {
+                this.Output(logEvent, this.Layout.Render(logEvent));
+            }
+            else
+            {
+                this.FastOutput(logEvent, this.Layout.Render(logEvent));
+            }
+
+        }
+
+        private void FastOutput(LogEventInfo logEvent, string render)
+        {
+
+            ConsoleRowHighlightingRule matchingRule = FindConsoleRowHighlightingRule(logEvent);
```

Comment:
This is waht is not being executed in the fast modus.

    for (int index = 0; index < this.WordHighlightingRules.Count; index++)
    {
        ConsoleWordHighlightingRule hl = this.WordHighlightingRules[index];
        message = hl.ReplaceWithEscapeSequences(message);
    }
    ColorizeEscapeSequences(..........);